tree/treeplayer/src/TDFUtils.cxx

```diff
@@ -77,48 +150,23 @@ ColumnName2ColumnTypeName(const std::string &colName, TTree *tree, TCustomColumn
             return l->GetTypeName();
          } else {
             // we do not know how to deal with this branch
-            throw std::runtime_error("TTree branch " + colName + " has both a leaf count and a static length. This is not supported.");
+            throw std::runtime_error("TTree branch " + colName +
+                                     " has both a leaf count and a static length. This is not supported.");
```

Comment:
formatting unrelated lines might make it harder to cherry-pick this commit -- but it's my fault for not formatting this line properly before 😅 I wonder why travis let this pass